app/src/main/java/com/lamdangtung/calculatorapp/MainActivity.kt

```diff
@@ -0,0 +1,292 @@
+package com.lamdangtung.calculatorapp
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.TextView
+import com.lamdangtung.calculatorapp.databinding.ActivityMainBinding
+import java.util.*
+
+class MainActivity : AppCompatActivity(), View.OnClickListener {
+    // Constants
+    private val EXTRA_TEXT_FORMUlA: String = "com.lamdangtung.calculatorapp.EXTRA_TEXT_FORMULA"
```

Comment:
```suggestion
    private val EXTRA_TEXT_FORMUlA = "com.lamdangtung.calculatorapp.EXTRA_TEXT_FORMULA"
```
đây là const nên ô có thể cho vào component object

Reply:
ý anh có phải là companion object không ?

Reply:
ừ đúng rồi :D